background+box+model/index.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html lang="pl">
+    <head>
+        <meta charset="UTF-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <link rel="stylesheet" href="../css/reset.css">
+        <link rel="stylesheet" href="../css/style-zadanie-5.css">
+        <title>My Osom Site</title>
+    </head>
+    
+    <body>
+        <header>
+            <nav>
+                <a class="logotype">Logo</a>
+                <a href="#">Home</a>
+                <a href="#">About</a>
+                <a href="#"">Contact</a>
+            </nav> 
+        </header>
+        <main>
+            <section class="slider">
+                <h1>My Osom Site</h1>
+            </section>
+            <section class="articles">
+               <article>
+                <h2>my first article</h2>
+                <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Suscipit, nihil magnam nostrum eveniet qui optio, magni dolores nam excepturi quia inventore sed. Totam dolorum facilis repellendus saepe iusto possimus aut!</p>
+                <p>20.02.2020</p>
+                </article> 
+                <div class="space"></div>
+                <article>
+                    <h2>my second article</h2>
+                    <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Suscipit, nihil magnam nostrum eveniet qui optio, magni dolores nam excepturi quia inventore sed. Totam dolorum facilis repellendus saepe iusto possimus aut!</p>
+                    <p>24.04.2020</p>
+                    </article> 
+            </section>
+            
+        </main>
```

Comment:
```suggestion
<main>
    <section class='slider'>
        <h1>My Osom Site</h1>
    </section>

    <section class='articles'>
        <article>
            <h2>my first article</h2>

            <p>
                Lorem ipsum dolor sit amet consectetur adipisicing elit. Suscipit, nihil magnam nostrum eveniet qui
                optio, magni dolores nam excepturi quia inventore sed. Totam dolorum facilis repellendus saepe iusto
                possimus aut!
            </p>

            <p>20.02.2020</p>
        </article>

        <div class='space'></div>

        <article>
            <h2>my second article</h2>

            <p>
                Lorem ipsum dolor sit amet consectetur adipisicing elit. Suscipit, nihil magnam nostrum eveniet qui
                optio, magni dolores nam excepturi quia inventore sed. Totam dolorum facilis repellendus saepe iusto
                possimus aut!
            </p>

            <p>24.04.2020</p>
        </article>
    </section>
</main>
```

Reply:
tu moja sugestia jakby to można było poformatować, ofc nie musisz sie tego sztywno trzymać, jak widzisz poodzielałam tutaj elementy blokowe, wizualnie takie coś się dużo łatwiej przegląda/czyta. Generalnie pewnie style formatowania w firmach sie róznia subtelnie, ale oddzielanie elementów blokowych jest moco powszechne :)